automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/SSH.scala

```diff
@@ -142,11 +142,24 @@ object SSH {
     } yield output
 
   // This method is the main one for any interaction with GCP VMs
-  def executeGoogleCommand(project: GoogleProject, zone: String, runtimeName: RuntimeName, cmd: String): IO[String] = {
+  def executeGoogleCommand(project: GoogleProject,
+                           zone: String,
+                           runtimeName: RuntimeName,
+                           cmd: String,
+                           useCmdDirectly: Boolean = false
+  ): IO[String] = {
     val dummyCommand =
       s"gcloud compute ssh --zone '${zone}' '${LeonardoConfig.GCS.leonardoServiceAccountUsername}@${runtimeName.asString}' --project '${project.value}' --tunnel-through-iap -q --command=\"ls\" -- -tt"
     val sshCommand =
-      s"gcloud compute ssh --zone '${zone}' '${LeonardoConfig.GCS.leonardoServiceAccountUsername}@${runtimeName.asString}' --project '${project.value}' --tunnel-through-iap -q --command=\"$cmd\" -- -tt"
+      if (!useCmdDirectly)
+        s"""
+          gcloud compute ssh --zone '${zone}' '${LeonardoConfig.GCS.leonardoServiceAccountUsername}@${runtimeName.asString}' --project '${project.value}' --tunnel-through-iap -q --verbosity=error --command="$cmd" -- -tt
```

Comment:
does it make sense to add something that properly escapes the `$cmd` for the caller? I am thinking of how much time we spent figuring out the right incantation, and it would be great to immortalize that information in the code somehow

Reply:
Hmm... I don't know if its possible to fully encapsulate it. I can surround `cmd` with single quotes, but the caller will still need to figure out how to escape any embedded quotes 😞

Reply:
Hmm... ok I have an idea based on your comment further down